test/alignment-D3-support.js

```diff
@@ -1,18 +1,7 @@
 // BDD tests for the alignment-D3-support module.
 
-// TODO: Most functions are not tested yet. The following functions need tests:
```

Comment:
I think the `getShapeValue` and `getTickValue` functions don't have tests yet. Could we leave this comment in and with those function listed as needing tests?

These tests could be also more robust. For example, `getShapePosition` supports 'topLeft' and 'center' for the `positionType`, but only 'center' is tested. That doesn't need to be done for this PR, but a comment noting that would be good so we don't forget.